apps/src/applab/applab.js

```diff
@@ -1040,7 +1041,11 @@ Applab.init = function(config) {
     }
     var designModeButton = document.getElementById('designModeButton');
     if (designModeButton) {
-      dom.addClickTouchEvent(designModeButton, Applab.onDesignModeButton);
+      // Simulate a run button click, to load the channel id.
+      var designModeClick = studioApp.runButtonClickWrapper.bind(
```

Comment:
This feels a little bit weird. Is there a better name for runButtonClickWrapper that makes it's purpose clearer (and makes it clearer that we're not just doing it on run button click)?